modules/serverless-invoking-go-functions.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-go-functions.adoc
+
+[id="serverless-invoking-go-functions_{context}"]
+= About invoking Golang functions
+
+Golang functions are invoked by using different methods, depending on whether they are triggered by a HTTP request or a CloudEvent.
+
+[id="serverless-invoking-go-functions-http_{context}"]
```

Comment:
These two sections (HTTP and CloudEvent) should probably be each in their own module. You could put the above heading and lead-in sentence in the assembly and then include the two modules.

Reply:
Split into two modules